discovery-provider/es-indexer/etl/jobRunner.ts

```diff
@@ -0,0 +1,148 @@
+import { Client } from '@elastic/elasticsearch'
```

Comment:
I think this file could be a bit more polished up (commented out code, any's, etc.)